app/models/post.rb

```diff
@@ -1,2 +1,5 @@
 class Post < ApplicationRecord
```

Comment:
- [ ] `Post` model should have method that returns the 5 most recent comments for a given post. This should be similar to the example below:

```ruby
# app/models/post.rb
...

def recent_comments
  comments.order('created_at Desc').limit(5)
end
```

Reply:
Done